data_centre/data.py

```diff
@@ -468,7 +468,7 @@ def get_display_modes_list(self, with_nav_mode=False):
             if self.settings['shader']['USE_SHADER_BANK']['value'] == 'enabled' and ["SHADERS",'NAV_SHADERS'] in display_modes:
                 display_modes.append(["SHDR_BNK",'PLAY_SHADER'])
             if self.settings['shader']['USE_SHADER_MOD']['value'] == 'enabled' and ["SHADERS",'NAV_SHADERS'] in display_modes:
-                display_modes.append(["SHDR_MOD","NAV_MOD"]) ## allow override, but fall back to PLAY_SHADER controls
+                display_modes.append(["SHDR_MOD",["NAV_MOD","PLAY_SHADER"]]) ## allow override, but fall back to PLAY_SHADER controls
```

Comment:
@doctea does the display handle this list now ?

Reply:
dw i will fix it :)

Reply:
I hadn't noticed any problem with it myself -- what have I missed?